main/core/messagebroker.ts

```diff
@@ -175,6 +244,40 @@ export class MessageBroker<T = any> implements IMessageBroker<T> {
         };
     }
 
+    private getOrCreateAdapterObservables<K extends keyof T>(channelName: K): Observable<IMessage<T[K]>>[] {
+        if (!this.adapterObservables[channelName]) {
+            this.adapterObservables[channelName] = {};
+        }
+
+        const channelAdapterObs = this.adapterObservables[channelName]!;
```

Comment:
I find a neater way of doing this is:

```
const channelAdapterObs = this.adapterObservables[channelName] ?? (this.adapterObservables[channelName] = {});
```

does it all in one line with no need for a bang `!`